modules/thermal_hydraulics/test/tests/problems/pressure_drop/pressure_drop.i

```diff
@@ -0,0 +1,138 @@
+nelem = 100
+friction_factor = 1e4
+
+area = 0.176752
+mfr_final = 1.0
+p_out = 7e6
+T_in = 300
+ramp_time = 5.0
+
+[GlobalParams]
+  gravity_vector = '0 0 0'
+  initial_T = ${T_in}
+  initial_p = ${p_out}
+  initial_vel = 0
+  closures = no_closures
+  rdg_slope_reconstruction = full
+  scaling_factor_1phase = '1 1 1e-5'
+[]
+
+[Modules]
+  [FluidProperties]
+    [h2]
+      type = IdealGasFluidProperties
+      gamma = 1.3066
+      molar_mass = 2.016e-3
+      k = 0.437
+      mu = 3e-5
+    []
+  []
+[]
+
+[Closures]
+  [no_closures]
+    type = Closures1PhaseNone
```

Comment:
Is there a reason not to use the simple closures? the friction factor is constant.

Reply:
Nope, you were right. I don't know why I did that.